vitest.setup.ts

```diff
@@ -79,6 +79,8 @@ const originalLocalStorageGetItem = localStorage.getItem;
 let localstorageSpy;
 
 beforeAll(async () => {
```

Comment:
https://github.com/ArdentHQ/arkvault/pull/523/files#diff-0a7996d580fb7938335496ed892e3c0a70af5770fdce11c375de28b9ec7b43cdR92

Got the same error and applied the given fix.